lib/diagnostic.rb

```diff
@@ -106,10 +106,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = [age: 20, first_name: "Louis"]'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person[:first_name].default = "He is pretty confused."'
```

Comment:
This will cause an error.  Assuming that `person` is properly defined, `person[:first_name]` will be a string and `String` doesn't have a `default` method.